lib/danger/request_sources/github.rb

```diff
@@ -50,7 +50,7 @@ def update_pull_request!(warnings: nil, errors: nil, messages: nil)
         # Just remove the comment, if there's nothing to say.
         delete_old_comments!
       else
-        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id)
+        issues = client.issue_comments(ci_source.repo_slug, ci_source.pull_request_id, auto_pagination: true)
```

Comment:
AFAIK it's possible to set this globally, so we don't have to do this for each request